crest/Assets/Crest/Crest/Scripts/LodData/RegisterHeightInput.cs

```diff
@@ -73,30 +71,4 @@ protected override void Update()
         protected override bool FeatureEnabled(OceanRenderer ocean) => true;
```

Comment:
yeah probably worth it. just to confirm, does this mean adding code somewhere that will calculate maxy/miny and then max disp, like those lines above? i'm guessing that will need to iterate over spline points and take max/min y? it would need to do it once on awake, and then maybe if UNITY_EDITOR do it every update. does that sound right?

Reply:
nvm i found a way to piggy back off existing stuff and it seems to work

Reply:
Thanks. Works well.